crates/lang/src/lib.rs

```diff
@@ -58,6 +58,7 @@ pub use self::{
     events::{
         BaseEvent,
         EmitEvent,
+        PrefixedValue,
```

Comment:
We should not publicly make this available since I consider it to be kind of an implementation detail.